tfprotov5/internal/fromproto/raw_identity.go

```diff
@@ -1,20 +0,0 @@
-// Copyright (c) HashiCorp, Inc.
-// SPDX-License-Identifier: MPL-2.0
-
-package fromproto
-
-import (
-	"github.com/hashicorp/terraform-plugin-go/tfprotov5"
-)
-
-func RawIdentity(in []byte) *tfprotov5.RawIdentity {
```

Comment:
Since there is no longer a concept of a `tfprotov5.RawIdentity`, can we delete the struct/tests as well?

At `tfprotov{5/6}/raw_identity.go`

Reply:
here's a PR with that: https://github.com/hashicorp/terraform-plugin-go/pull/489